### DISCUSSION TITLE
Hard fail to suppressionslist

### DISCUSSION THREAD

**Discussion by sinnbeck** [postalserver/postal#2497] [Feature suggestions]
Currently it seems that the only way a mail can end up on the suppressions list is to have many soft-fails. But I would like to have all hard-fails end up on the suppression list directly. This would probably require us to be able to remove them from the list via the GUI as well. I would expect this to be a config value as I assume not everyone want this.

This will ensure we dont accidental send to the same non-existent user multiple times and thereby risk getting our ip blacklisted.

**Comment by willpower232** [comment]
I think you could experiment with reducing this value https://github.com/postalserver/postal/blob/8b525d0381a9e0113af808b9ec2eb47bf78ec60b/config/postal.defaults.yml#L14

**Reply by sinnbeck** [reply]
If the recipient does not exist it fails in 1 try (but not go on the suppression list). This only applies to soft-fail :)
